README.md

```diff
@@ -7,3 +7,11 @@ Kombu Transport using Redis SortedSets
 ## Running tests
 
     python setup.py test
+
+## Using with celery tasks - zpriority
+
+    task.apply_async(zpriority=10)
```

Comment:
That's pretty crazy that you can straight pass through celery a new option like this.